core/dateUtils.js

```diff
@@ -108,10 +108,22 @@ export const isValidDateInFormat = (dateStr, format) => {
   return fnsIsValid(parsed)
 }
 
+const substractTimezoneOffset = (date) => {
+  const timezoneOffset = date.getTimezoneOffset() * 60000
```

Comment:
This and 117 could be extracted to use the 60000 constant in only one place

Reply:
right